index.js

```diff
@@ -0,0 +1,127 @@
+    source: 'https://github.com/God-Logic/My-Portfolio-Website',
+  },
+  {
+    image: '../images/Card 2.png',
+    imageHover: '../images/backround image.png',
+    name: 'Multi-Post Stories',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+  {
+    image: '../images/Card 2.png',
+    imageHover: '../images/backround image.png',
+    name: 'Multi-Post Stories',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+];
+window.onload = () => {
+  const menu = document.querySelector('.hamburger-menu');
+  const close = document.querySelector('.close-icon');
+  const workLink = document.querySelector('.work-link');
+  const aboutLink = document.querySelector('.about-link');
+  const contactLink = document.querySelector('.contact');
+  menu.onclick = () => {
+    document.querySelector('.mobile-menu').classList.remove('hide-menu');
+    document.querySelector('.mobile-menu').classList.add('show-menu');
+  };
+  close.onclick = () => {
+    document.querySelector('.mobile-menu').classList.add('hide-menu');
+  };
+  workLink.onclick = () => {
+    document.querySelector('.mobile-menu').classList.add('hide-menu');
+  };
+  aboutLink.onclick = () => {
+    document.querySelector('.mobile-menu').classList.add('hide-menu');
+  };
+  contactLink.onclick = () => {
+    document.querySelector('.mobile-menu').classList.add('hide-menu');
+  };
+
+  const card = (data) => `<div class="alt-card" style=background-repeat:no-repeat;background-size:cover;">
+    <h2>${data.name}</h2>
+    <p>${data.description}</p>
+    <ul>
+      <li>${data.technologies[0]}</li>
+      <li>${data.technologies[1]}</li>
+      <li>${data.technologies[2]}</li>
+    </ul>
+    <button class="see-project">See Project</button>
+  </div>`;
+
+  const container = document.querySelector('.middle-part');
+  container.innerHTML = datas.map((data) => card(data)).join('');
+  const altCards = document.querySelectorAll('.alt-card');
+
+  const button = document.querySelector('.see-project');
+  altCards.forEach((altCard) => {
+    altCard.onmouseover = () => {
+      button.style.opacity = '1';
+    };
+
+    altCard.onmouseout = () => {
+      button.style.opacity = '0';
+    };
+  });
+
+  const popup = document.querySelector('#popup');
+  const openPopup = () => {
+    popup.style.display = 'block';
+  };
+
+  container.addEventListener('click', (event) => {
+    if (event.target.matches('.see-project')) {
+      openPopup();
+    }
+  });
+
+  button.onclick = () => {
+    popup.style.display = 'block';
+  };
```

Comment:
- [ ] kindly consider making sure that if user clicks (or taps) the button to check project details, the popup which appear follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) and kindly put in consideration to make your popups responsive on both mobile and desktop screens to implement exactly whats given in the figma design  check on the images below for more clarification 👇 
- your design 👇 
<img width="1361" alt="Screen Shot 2023-04-13 at 23 36 14" src="https://user-images.githubusercontent.com/60146030/231888360-c09e6909-d77b-465a-9073-4f44bb08ddca.png">

- provided design 👇 
<img width="835" alt="Screen Shot 2023-04-13 at 23 38 11" src="https://user-images.githubusercontent.com/60146030/231888650-08c5c57c-57cc-4361-abab-dee412a31d9f.png">